Makefile

```diff
@@ -83,15 +83,16 @@ clean-tests:
 #------------------------------------------------------
 # Run Targets
 #------------------------------------------------------
+CP ?= old
 unit-tests:
 	@echo "Running unit tests..."
 	$(GO) test -v -count=1 ./pkg/...  -json -cover | tparse --all
 
-tests-e2e: clean-tests 	docker-build 
-	$(GO) test -p 1 -timeout 30m -v -tags e2e ./tests/e2e/connectivity/...
+tests-e2e: build docker-build
+	$(GO) test -p 1 -timeout 30m -v -tags e2e ./tests/e2e/connectivity/... --controlplane=$(CP)
 
-tests-iperf3: clean-tests 	docker-build 
-	$(GO) test -p 1 -timeout 30m -v -tags e2e ./tests/e2e/iperf3/...
+tests-iperf3: build docker-build 
```

Comment:
Can we merge the tests? tests-e2e to run iperf tests too
Otherwise, we would be performing build redundantly if we are running both the tests.

Reply:
So why not use only the iperf3 test?

Reply:
Using iperf3 for just testing connectivity and policy test seems like an overkill. I thought iperf3 was more for benchmarking performance, which we want to do part of each test. My problem is with having to build each time for running a test especially if both tests are going to be run.  Ideally, we can use the connectivity test to test policies, load balancing, etc in future.. iperf3 for only performance benchmarking.